### DISCUSSION TITLE
[FEATURE] Add the option to add specific members to affiliate plan

### DISCUSSION THREAD

**Discussion by Jimster4312** [Paymenter/Paymenter#335] [Ideas]
Instead of enabling affiliates for all users be able to set it at only enabled for certain users.
Either doing this by selecting them in the affiliates settings.
Or by opening a client profile and selecting enable affiliates.

**Comment by finnjosten** [comment]
Or just both ways

**Reply by Jimster4312** [reply]
Yea could do, like the option for in client profile and for selecting them in the affiliates Settings
